rest/model/distro.go

```diff
@@ -157,7 +157,7 @@ func (apiDistro *APIDistro) BuildFromService(h interface{}) error {
 	apiDistro.Name = ToAPIString(d.Id)
 	apiDistro.UserSpawnAllowed = d.SpawnAllowed
 	apiDistro.Provider = ToAPIString(d.Provider)
-	if d.ProviderSettings != nil && (d.Provider == evergreen.ProviderNameEc2Auto || d.Provider == evergreen.ProviderNameEc2OnDemand || d.Provider == evergreen.ProviderNameEc2Spot) {
+	if d.ProviderSettings != nil && (d.Provider == evergreen.ProviderNameEc2Auto || d.Provider == evergreen.ProviderNameEc2OnDemand || d.Provider == evergreen.ProviderNameEc2Spot || d.Provider == evergreen.ProviderNameEc2Fleet) {
```

Comment:
can you make a helper function for the bits of this after the &&

Reply:
done.